composer.json

```diff
@@ -43,7 +43,8 @@
         "burgov/key-value-form-bundle": "to edit assoc multivalue properties. require version 1.0.*"
     },
     "conflict": {
-        "phpcr/phpcr-shell": "<1.0.0-beta1"
+        "phpcr/phpcr-shell": "<1.0.0-beta1",
+        "symfony/framework-bundle": "<2.3.27|>=2.6.0,<2.6.6"
```

Comment:
Honestly, I don't know if that's intended. That's just what I guess based on the initial constraint in the  `require` section.